To be merged 2.8.0 - MTV-2033 RN 2.8.0 (#655)
kubev2v/forklift-documentation: documentation/modules/known-issues-2-8.adoc

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

[id="known-issues-2-8_{context}"]
= Known issues

{project-first} 2.8 has the following known issues:

> .Trusted Platform Module is added to BIOS-based Windows Server 2022 VMs after warm migration

Reviewer: ```suggestion
.Trusted Platform Module is added to the BIOS of a Windows Server 2022 VM after warm migration
```
not too sure if we should use `BIOS-based`

> After warm migration of Windows Server 2022 virtual machine (VM) from vSphere to an {ocp-name} cluster, the Trusted Platform Module (TPM) device is added in the target VM, even if the TPM configuration is non-persistent in the BIOS setting of the source Windows Server 2022 VM. This prevents the target VM from starting. link:https://issues.redhat.com/browse/MTV-2049[(MTV-2049)]

Reviewer: ```suggestion
After warm migration of Windows Server 2022 virtual machine (VM) from vSphere to an {ocp-name} cluster, the Trusted Platform Module (TPM) device is added to the target VMs, even if the TPM configuration is non-persistent in the BIOS setting of the source Windows Server 2022 VM. This prevents the target VM from starting. link:https://issues.redhat.com/browse/MTV-2049[(MTV-2049)]
```

> .Linux virtual machines with clustered file systems are not supported

Reviewer: ```suggestion
.Linux VMs with clustered file systems are not supported
```  
as you used `VMs` in the previous header, i would use the same pattern here

> Linux guest VMs with clustered file systems on a shared disk are not supported. The `virt-v2v` tool may not update shared disks in the `fstab` configuration file. link:https://issues.redhat.com/browse/MTV-2145[(MTV-2145)]

Reviewer 2: Can we say "does not update" rather than "may"?

> Linux guest virtual machines with clustered file systems on a shared disk are not supported. The virt-v2v tool may not update shared disks in the `fstab`configuration file. link:https://issues.redhat.com/browse/MTV-2145[(MTV-2145)]

Reviewer 3: ```suggestion
Linux guest virtual machines with clustered file systems on a shared disk are not supported. The virt-v2v tool may not update shared disks in the `fstab` configuration file. link:https://issues.redhat.com/browse/MTV-2145[(MTV-2145)]
```

Author: Thank you @yaacov , made the change in the document.

> Linux guest virtual machines with clustered file systems on a shared disk are not supported. The virt-v2v tool may not update shared disks in the `fstab` configuration file. link:https://issues.redhat.com/browse/MTV-2145[(MTV-2145)]

Reviewer: ```suggestion
Linux guest VMs with clustered file systems on a shared disk are not supported. The virt-v2v tool may not update shared disks in the `fstab` configuration file. link:https://issues.redhat.com/browse/MTV-2145[(MTV-2145)]
```

as you have explained the acronym in line 11: `...virtual machine (VM)... `
i think you are ok to use the acronym here

// Module included in the following assemblies:
//
// * documentation/doc-Release_notes/master.adoc

[id="known-issues-2-8_{context}"]
= Known issues

{project-first} 2.8 has the following known issues:

> .Target VMware virtual machines lack connectivity because of unassigned static IP address

Reviewer 4: This issue got fixed. No need for a known issue.

After migrating Ubuntu 20.04 VMware virtual machines (VM) to {virt} 4.15.6 using {project-short}, the  network interface card (NIC) name of the source VM is not preserved. This results in a lack of network connectivity for the VM because the network interface gets a new name post migration. The static IP address that was configured in the network plan before migration is not assigned to the new network interface. link:https://issues.redhat.com/browse/MTV-1607[(MTV-1607)]

> .Persistent volume claims are populated by incorrect disks

Reviewer 4: This also got fixed in 2.8. It can be removed
